internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -441,6 +442,26 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 		c.vsock = h.devNullTransport
 	}
 
+	logPath := settings.OCISpecification.Annotations[annotations.LCOWTeeLogPath]
+	if logPath != "" {
+		if !allowStdio {
+			return nil, errors.Errorf("teeing container stdio to log path %q denied due to policy not allowing stdio access", logPath)
+		}
+
+		if logPath, err = filepath.Abs(logPath); err != nil {
+			return nil, errors.Wrapf(err, "failed to evaluate log path: %s", logPath)
+		}
+
+		log.G(ctx).WithField(logfields.Path, logPath).Debug("creating container log file")
+		dir := filepath.Dir(logPath)
+		if err := mkdirAllModePerm(dir); err != nil {
+			return nil, errors.Wrapf(err, "failed to create log file parent directory: %s", dir)
+		}
+		if c.logFile, err = os.Create(logPath); err != nil {
```

Comment:
doesn't look like this file is ever deleted? Is it the desired behavior?
Is it ok that when a container with the same config is created, the old log file is nuked?